doc/source/cluster/cluster_under_construction/ray-clusters-on-kubernetes/index.md

```diff
@@ -15,9 +15,6 @@ different Ray versions in the same Kubernetes cluster.
 ```{eval-rst}
 .. image:: /cluster/cluster_under_construction/ray-clusters-on-kubernetes/images/ray_on_kubernetes.png
     :align: center
-
-..
-  https://docs.google.com/drawings/d/1E3FQgWWLuj8y2zPdKXjoWKrfwgYXw6RV_FWRwK8dVlg/edit
```

Comment:
wait, why remove this? This is supposed to be a comment with the underlying google docs for the graphics so we can modify it in the future :)

Reply:
Ah, I see -- let me put that back in (with a note on the purpose)

Reply:
Reinserted.